### PR TITLE
Add additional fields to offer space type page

### PR DESCRIPTION
**Pull request by bilbof**
Now a description field is required for each type of space that is offered.
There is also a general description field that is required.

https://trello.com/c/3UKYWxkd/7-business-form-content-change-size-of-space

Before:

Screenshot 2020-04-03 at 11 48 28

After:

Screenshot 2020-04-03 at 11 49 15

Screenshot 2020-04-03 at 11 28 55

Screenshot 2020-04-03 at 11 28 39

### CODE REVIEW

File: app/views/coronavirus_form/offer_space_type.erb [hunk 1]
```diff
@@ -24,27 +24,57 @@
...
       },
       {
         value: t('coronavirus_form.questions.offer_space_type.options.other.label'),
         label: t('coronavirus_form.questions.offer_space_type.options.other.label'),
         checked: session.fetch(:offer_space_type, []).include?(t('coronavirus_form.questions.offer_space_type.options.other.label')),
-        conditional: render("govuk_publishing_components/components/textarea", {
+        conditional: render("govuk_publishing_components/components/input", {
           label: {
-            text: t('coronavirus_form.questions.offer_space_type.options.other.input.label'),
+            text: t('coronavirus_form.questions.offer_space_type.options.office_space.description.label'),
```
Comment (brucebolt) [inline app/views/coronavirus_form/offer_space_type.erb]:
```suggestion
            text: t('coronavirus_form.questions.offer_space_type.options.other.description.label'),
```

**Review by brucebolt — CHANGES_REQUESTED** [review]
Looks good 👍.  There's just one field label which is using an incorrect internationalisation.

**Comment by bilbof** [comment]
@brucebolt thanks! I've committed that change.

**Review by brucebolt — APPROVED** [review]
🎉

**Pull request merged**